model/task/task.go

```diff
@@ -1382,15 +1385,10 @@ func (t *Task) GetDisplayStatus() string {
 		if !t.Activated {
 			return evergreen.TaskUnscheduled
 		}
-		dependenciesMet, err := t.DependenciesMet(map[string]Task{})
```

Comment:
removed this check because it would query the db for every single dependent task. Instead we should just rely on the unattainable field in the task since that should represent if a task is blocked or not.